src/system.jl

```diff
@@ -73,8 +73,8 @@ end
 """
     $TYPEDEF
 
-Type for static branch attributes.  Branches may have between 0 and 2 break points
-which is why the `break_points` and `penalties` fields contain variable length `Tuple`s.
+Type for static non-transformer branch attributes.  Branches may have between 0 and 2 break
```

Comment:
isn't this the opposite for what the PR is doing, which is adding _transformer_ branch attributes?
```suggestion
Type for static branch attributes.  Branches may have between 0 and 2 break
```